Add information to readme (#25)
dwp/dataworks-aws-ingestion-ecs-cluster: README.md

    1. Add/remove jobs in `./ci/jobs` as required 
    1. Create CI pipeline:  
`aviator`
       
## Infrastructure

> This repo builds an AWS ECS cluster named `ingestion` for which the `claimant-api-kafka-consumer` application operates upon.

Reviewer: ```suggestion
This repo builds an AWS ECS cluster named `ingestion` onto which the `claimant-api-kafka-consumer` application is deployed.
```
